examples/get_transactions.py

```diff
@@ -0,0 +1,73 @@
+import os
+import sys
+import csv
+import getpass
+import StringIO
+import argparse
+
+from collections import defaultdict
+
+# Fix for CERTIFICATE_VERIFY_FAILED error in python 2.7.9 (see https://bugs.python.org/issue22417)
+import ssl
+if hasattr(ssl, '_create_unverified_context'):
+    ssl._create_default_https_context = ssl._create_unverified_context
+
+# TODO: fix this
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+import mturkweb
+
+CONSIDER_PAYMENT_TYPES = ['AssignmentPayment', 'BonusPayment']
+
+
+def login_prompt():
+    email = raw_input('Email: ')
+    password = getpass.getpass('Password: ')
+    return email, password
+
+def parse_csv(csv_string):
+    results = defaultdict(lambda: [0,0])
+    csv_file = StringIO.StringIO(csv_string)
+    csvreader = csv.DictReader(csv_file, delimiter=',')
+    
+    for row in csvreader:
+        if row['Transaction Type'] in CONSIDER_PAYMENT_TYPES:
+            sign = 1 if row['Amount'][0] != '-' else -1
+            amount = float(row['Amount'].lstrip('-')) * sign
+            results[row['Recipient ID']][0] += amount
+            results[row['Recipient ID']][1] += 1
+            
+    csv_file.close()
+
+    sorted_results = sorted([[k, v[0], v[1]] for k, v in results.iteritems()], key=lambda x: x[1])
+    return sorted_results
+
+def main(argv):
+    parser = argparse.ArgumentParser(description='Download the transaction history from mturk')
+    
+    try:
+        parser.add_argument('year', help='The year you want the transaction history for')
+        parser.add_argument('--amount', help='Only print payments above this threshold (default: 10)', default=10)
+        
+        args = parser.parse_args(sys.argv[1:])
```

Comment:
On my system, this throws an `exceptions.SystemExit` that isn't caught. I suggest adding this before the call to `parse_args`:

        if len(sys.argv) == 1:
            parser.print_help()
            sys.exit(2)